sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -162,6 +164,70 @@ client = EventGridPublisherClient(endpoint, credential)
 
 client.send(event)
 ```
+
+### Send Multiple events
+
+It is possible to send events as a batch when sending multiple events to a topic or a domain. This example send a list of CloudEvents using the send method.
+
+**WARNING** To gain the best performance when sending multiple events at one time, it is highly recommended to send a list of events instead of iterating over and sending each event in a loop.
```

Comment:
I'm not sure WARNING is the best word to use here, since the sentence is not saying "if you do this, this bad result will happen". It's more of a **NOTE** maybe?

If you would like to stick with WARNING, maybe something closer to this?
```suggestion
**WARNING:** When sending a list of multiple events at one time, iterating over and sending each event will not result in optimal performance. For best performance, it is highly recommended to send a list of events.
```